multicluster/controllers/multicluster/member/node_controller.go

```diff
@@ -116,12 +119,15 @@ func (r *NodeReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.
 	var isValidGateway bool
 
 	if stillGatewayNode {
+		gw.ServiceCIDR, err = r.getServiceCIDR()
+		if err != nil {
+			klog.ErrorS(err, "Failed to discover Service CIDR in cluster")
```

Comment:
I suppose you should return error here  and retry since ServiceCIDR is a mandatory field.